packages/swapper/src/swappers/thorchain/types.ts

```diff
@@ -21,6 +21,21 @@ export type PoolResponse = {
   volume24h: string
 }
 
+type MidgardActionOut = {
+  txID: string
+}
+type MidgardAction = {
+  date: string
+  height: string
+  out: [MidgardActionOut]
```

Comment:
is this guaranteed to be an array of exactly length 0?

Reply:
this is not an opinion or stylistic - `[Type]` is not the same as `Type[]`

Reply:
ah yes good point. long day whoops

Reply:
`MidgardActionsResponse['actions']` has been updated but this is still there. Again, do we want this to be a tuple?